docs/konnectors.md

```diff
@@ -208,6 +208,114 @@ single thread or application. But NaCl is [no longer
+Note : the limitation of NsJail I saw might be due to my lack of knowledge regarding system administration.
+
+### Proposed use of rkt regarding connectors
+
+#### Installation
+
+As stated before, rkt is easy to install. It may also be possible to make it available in the cozy-stack docker image but I did not test it (TODO)
+
+#### Image creation
+
+To create an ACI file image, you just need to run a docker image one time :
+    
+    rkt run  --uuid-file-save=$PWD/uuid --insecure-options=image --interactive docker://node:slim --name nodeslim -- -v
+    rkt export --app=nodeslim `cat uuid` nodeslim.aci && rm uuid
+
+
+The node:slim image weights 84M at this time. The node:alpine image also exists and is way lighter (19M) but I had problems with DNS with this, which could be solved with more time.
+
+#### Running a connector
+
+A path dedicated to run the konnectors with a predefined list of node packages available (the net module could be mocked with special limitations to blacklist some urls)
+
+A script will run the node container giving as option the script to launch. The path is mounted inside the container. The following script does just that
+
+    #!/usr/bin/env bash
+    rm -rf ./container_dir
+    cp -r ./container_dir_template ./container_dir
+    rkt run --uuid-file-save=$PWD/uuid --volume data,kind=host,source=$PWD/container_dir --insecure-options=image nodeslim.aci --cpu=100m --memory=128M --name rktnode --mount volume=data,target=/usr/src/app --exec node -- /usr/src/app/$1 &
+    sleep 60   # 1 min seems enough to run a connector
+    rkt stop --force --uuid-file=uuid
+    rkt rm --uuid-file=uuid
+    # read needed information in the container_dir directory : new documents, logs, etc
+    rm -rf ./container_dir
+
+This script can be run like this :
+
+    ./rkt.sh mynewkonnector.js
+
+If the mynewkonnector.js file is available in the container_dir_template directory.
+
+Pros : no breach regarding localhost network, more secure
+Cons: a file communication protocol with the stack need to be defined (any other idea ?), it may be more difficult to migrate the existing connectors.
+
+If we choose to give access to the host network to the container (but with a limited mocked net module maybe), the solution will be slightly modified : 
+
+    #!/usr/bin/env bash
+    rm -rf ./container_dir
+    cp -r ./container_dir_template ./container_dir
+    rkt run --net=host --uuid-file-save=$PWD/uuid --volume data,kind=host,source=$PWD/container_dir --insecure-options=image nodeslim.aci --cpu=100m --memory=128M --name rktnode --mount volume=data,target=/usr/src/app --exec node -- /usr/src/app/$1 $2 &
+    # the container will handle itself the communication with the stack
+    sleep 60
+    rkt stop --force --uuid-file=uuid
+    rkt rm --uuid-file=uuid
+    rm -rf ./container_dir
+
+This script can be run like this :
+
+    ./rkt.sh mynewkonnector.js COZY_STACK_CREDENTIALS
```

Comment:
The credentials must be passed in env variables, not on the command-line. Else, they appear on `ps aux` and can be easily stolen by another user on the server.